bin/p2-shutdown/main_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/square/p2/pkg/types"
+)
+
+func TestShutdownPod(t *testing.T) {
+	testCases := []struct {
+		pod            types.PodID
+		podsToShutdown []types.PodID
+		podsToExclude  []string
+		expectation    bool
+	}{
+		{types.PodID("a"), []types.PodID{"a"}, []string{}, true},
+		{types.PodID("a"), []types.PodID{}, []string{"a"}, false},
+		{types.PodID("a"), []types.PodID{}, []string{}, true},
```

Comment:
maybe add a case where "b" is in teh shutdown list which should make the expectation for "a" false right?